cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -231,12 +233,7 @@ public BodyConsumer createAppVersion(HttpRequest request, HttpResponder responde
                                        @PathParam("version-id") final String versionId)
     throws Exception {
 
-    ApplicationId applicationId = validateApplicationVersionId(namespaceId, appId, versionId);
-
-    if (!applicationLifecycleService.updateAppAllowed(applicationId)) {
-      responder.sendString(HttpResponseStatus.CONFLICT,
-                           String.format("Cannot update the application because version %s already exists", versionId));
-    }
+    ApplicationId applicationId = validateApplicationVersionId(namespaceId, appId, UUID.randomUUID().toString());
```

Comment:
Can we just call the `create` method above instead? This makes sure the implementation are the same.